ppc regression

```
CC      arch/PowerPC/PPCDisassembler.o
[874](https://github.com/radareorg/radare2/actions/runs/5091297618/jobs/9151131957?pr=21789#step:3:875)
  CC      arch/PowerPC/PPCModule.o
[875](https://github.com/radareorg/radare2/actions/runs/5091297618/jobs/9151131957?pr=21789#step:3:876)
In file included from /usr/include/string.h:495,
[876](https://github.com/radareorg/radare2/actions/runs/5091297618/jobs/9151131957?pr=21789#step:3:877)
                 from arch/PowerPC/PPCInstPrinter.c:21:
[877](https://github.com/radareorg/radare2/actions/runs/5091297618/jobs/9151131957?pr=21789#step:3:878)
In function ‘strncpy’,
[878](https://github.com/radareorg/radare2/actions/runs/5091297618/jobs/9151131957?pr=21789#step:3:879)
    inlined from ‘PPC_printInst’ at arch/PowerPC/PPCInstPrinter.c:648:2:
[879](https://github.com/radareorg/radare2/actions/runs/5091297618/jobs/9151131957?pr=21789#step:3:880)
/usr/include/bits/string_fortified.h:106:10: error: ‘__builtin___strncpy_chk’ output may be truncated copying between 0 and 31 bytes from a string of length 511 [-Werror=stringop-truncation]
[880](https://github.com/radareorg/radare2/actions/runs/5091297618/jobs/9151131957?pr=21789#step:3:881)
  106 |   return __builtin___strncpy_chk (__dest, __src, __len, __bos (__dest));
[881](https://github.com/radareorg/radare2/actions/runs/5091297618/jobs/9151131957?pr=21789#step:3:882)
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[882](https://github.com/radareorg/radare2/actions/runs/5091297618/jobs/9151131957?pr=21789#step:3:883)
  CC      arch/PowerPC/PPCMapping.o
[883](https://github.com/radareorg/radare2/actions/runs/5091297618/jobs/9151131957?pr=21789#step:3:884)
  CC      arch/RISCV/RISCVModule.o
[884](https://github.com/radareorg/radare2/actions/runs/5091297618/jobs/9151131957?pr=21789#step:3:885)
  CC      arch/RISCV/RISCVMapping.o
```

--- a/arch/PowerPC/PPCInstPrinter.c
+++ b/arch/PowerPC/PPCInstPrinter.c
@@ -645,7 +645,7 @@ void PPC_printInst(MCInst *MI, SStream *O, void *Info)
 		mnem_len = sizeof(MI->flat_insn->mnemonic) - 1;
 
 	memset(MI->flat_insn->mnemonic, 0, sizeof(MI->flat_insn->mnemonic));
-	strncpy(MI->flat_insn->mnemonic, O->buffer, mnem_len);
+	memcpy(MI->flat_insn->mnemonic, O->buffer, mnem_len);
 }
 
 // FIXME

